cli/translate.py

```diff
@@ -0,0 +1,88 @@
+import sys
+from typing import Any, Dict, List
+
+from datastore.reader.core import FilterRequest, Reader
+from datastore.reader.services import register_services as register_reader_services
+from datastore.shared.di import injector
+from datastore.shared.util import FilterOperator, fqid_from_collection_and_id
+from datastore.writer.core import RequestUpdateEvent, Writer, WriteRequest
+from datastore.writer.services import register_services as register_writer_services
+
+from openslides_backend.locale.translator import Translator
+
+collection_to_fields_map = {
```

Comment:
Add the fields mentioned in https://github.com/OpenSlides/openslides-backend/pull/1502#issuecomment-1441767944

Reply:
Done. Added the fields to the `collection_to_fields_map`.